frontend/catalyst/api_extensions/quantum_operators.py

```diff
@@ -392,16 +394,28 @@ class MidCircuitMeasure(HybridOp):
 
     binder = qmeasure_p.bind
 
+    def __init__(self, *args, **kwargs):
+        self.bypass_postselect = kwargs.pop("bypass_postselect", False)
+        self.postselect = kwargs.pop("postselect", None)
+        self.reset = kwargs.pop("reset", False)
+        super().__init__(*args, **kwargs)
```

Comment:
Just a heads up that this will have overlapp with Mudit's #779 PR

Reply:
I can rebase that PR to use this one as the base so that we can integrate more easily.

Reply:
Let's hold off for now and then rebase. The requirements might be clearer once this PR goes in.